cmd/influx/task.go

```diff
@@ -53,3 +55,312 @@ func newTaskCreateCmd() *cli.Command {
 		},
 	}
 }
+
+func newTaskFindCmd() *cli.Command {
+	var params task.FindParams
+	params.Limit = TaskMaxPageSize
+	flags := append(commonFlags(), getOrgFlags(&params.OrgParams)...)
+	flags = append(flags, []cli.Flag{
+		&cli.StringFlag{
```

Comment:
Non-blocking since it's already inconsistent across commands in this repo, but curious to hear your thoughts:

It looks like these are `platform.ID`s on the backend. Stuart ported that type into this repo as `influxid.ID` so we could push the validation to the "edge" of CLI parsing ([for example](https://github.com/influxdata/influx-cli/blob/main/cmd/influx/org.go#L65-L71)). I've been trying to weigh the cost-benefit of that when all the CLI does with IDs is call `.String()` on them. What do you think?

Reply:
I'm a bit worried that if we every changed things the CLI would stop working (e.g. we change the ID length, allow a special value of some kind, etc) that would be hard to update.

I would be much happier if we were able to validate it at the openapi spec level - can we constrain and check fields on the API to ensure they are id's?

Reply:
Probably, if it's just a length limit & set of allowable characters... I can investigate